torch/testing/_internal/faulty_rpc_agent/faulty_process_group_agent_pybind.cpp

```diff
@@ -0,0 +1,80 @@
+#include <c10d/ProcessGroup.hpp>
+#include <caffe2/torch/testing/_internal/faulty_rpc_agent/faulty_process_group_agent.h>
+#include <pybind11/chrono.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+PYBIND11_MODULE(faulty_process_group_agent_pybind, m) {
+  m.doc() = "Module for Faulty Process Group RPC Agent.";
+
+  py::module rpc_module = py::module::import("torch.distributed.rpc");
```

Comment:
Can we put this under some private package like `torch.testing._internal.distributed.rpc`? This agent shouldn't be easily visible to users.

Reply:
I thought this `py::module::import` line is to import the RPC agent for use in this pybind file?